blobColor: 255
debug_view: false
filterByArea: true
filterByCircularity: true
filterByColor: true
filterByConvexity: true
filterByInertia: true
highHue: 179.0
highSat: 255.0
highVal: 255.0
image:
  compressed:
    format: jpeg
    jpeg_quality: 80
    png_level: 9
  compressedDepth:
    depth_max: 10.0
    depth_quantization: 100.0
    png_level: 9
  theora:
    keyframe_frequency: 64
    optimize_for: 1
    quality: 31
    target_bitrate: 800000
lowHue: 80.55000000000001
lowSat: 0.0
lowVal: 0.0
maxArea: 9000000
maxCircularity: 1.0
maxConvexity: 1.0
maxInertiaRatio: 1.0
minArea: 5000
minCircularity: 0.0
minConvexity: 0.0
minDistBetweenBlobs: 100
minInertiaRatio: 0.0
morphology_ex_image:
  compressed:
    format: jpeg
    jpeg_quality: 80
    png_level: 9
  compressedDepth:
    depth_max: 10.0
    depth_quantization: 100.0
    png_level: 9
  theora:
    keyframe_frequency: 64
    optimize_for: 1
    quality: 31
    target_bitrate: 800000
morphology_ex_kernel_size: 16
morphology_ex_type: opening
num_worker_threads: 1
queue_size: 3
thresholded_image:
  compressed:
    format: jpeg
    jpeg_quality: 80
    png_level: 9
  compressedDepth:
    depth_max: 10.0
    depth_quantization: 100.0
    png_level: 9
  theora:
    keyframe_frequency: 64
    optimize_for: 1
    quality: 31
    target_bitrate: 800000
use_camera_info: false
